md/migrate-from-an-earlier-version-of-bonita-bpm.md

```diff
@@ -298,13 +301,50 @@ please follow this procedure:
 
 ## Migrating to Bonita 7.9+ using Oracle
 
-Bonita 7.9+ supports Oracle 12c (12.2.x.y) version. To migrate to Bonita 7.9+ when using Oracle,
-please follow this procedure:
-* ensure your Bonita platform is shut down
-* in a first step, run Bonita migration tool to update Bonita platform to version 7.8.4, following the procedure above
-* then upgrade your Oracle database server to the version 12c (it must be 12.2.x.y)
-* in a second step, run the migration tool again to migrate the platform to 7.9.0 or newer
-* once done, you can restart your updated Bonita platform
+Bonita 7.9+ supports Oracle 12c (12.2.0.x.y) and Oracle 19c (19.3.0.0) versions: this is a requirement change.
+
+The Oracle database server change needs to be done before using the Bonita migration tool from 7.8.4 to 7.9.0.
+
+### Migrate to 7.8.4
+
+Skip this section and jump directly to **Upgrade Oracle database server** section if the 7.8.4 is already the version in use.
+
+* shut down the Bonita platform 
+* run Bonita migration tool to update Bonita platform to version 7.8.4, following the migration procedure [above](#migrate)
+
+### Upgrade Oracle database server
+
+* shut down the Bonita platform
+* upgrade the Oracle database server to the version 12c (it must be 12.2.0.x.y) or 19c (it must be 19.3.0.0)
+
+### Configure the Oracle database server
+
+* configure the Oracle database server, in particular activate the XA transactions management: see the *Oracle Database* section in the [Database creation and configuration for Bonita engine and BDM](database-configuration) page
+* install the missing Oracle components
+* execute the SQL scripts to *install* XA management elements
+* execute the SQL requests to GRANT the proper rights to the Oracle users; for both Bonita BPM and BDM schemas
+
+### Download the specific jdbc driver for the Oracle 12c (12.2.0.x.y) or 19c (19.3.0.0)
+
+**Beware**: two different jdbc driver jar files may share the same name (ojdbc8.jar for example).
+
+Each file however is specific to the Oracle DB server version installed.
+Please make sure to download the appropriate one:
+* Oracle 12c (12.2.0.x.y) : [Oracle Database 12.2.0.1 JDBC Driver & UCP Downloads](https://www.oracle.com/database/technologies/jdbc-ucp-122-downloads.html)
+* Oracle 19c (19.3.0.0) : [Oracle Database 19c (19.3) JDBC Driver & UCP Downloads](https://www.oracle.com/database/technologies/appdev/jdbc-ucp-19c-downloads.html)
```

Comment:
@jjargot please note that ojdbc8-19.3.0.0.jar is now provided in the migration tool.
Can you update this PR accordingly?